openlibrary/templates/lib/not_logged.html

```diff
@@ -1,3 +1,3 @@
 <div id="noLogin">
-    <p class="collapse sansserif small brown">$:_('<strong>You are not <a href="/account/login" title="Log in now">logged in</a>.</strong> Open Library will record your IP address and include it in this page\'s publicly accessible edit history. If you <a href="/account/create" title="Create an Open Library account">create an account</a>, your IP address is concealed and you can more easily keep track of all your edits and additions.')</p>
+    <p class="collapse sansserif small brown"><strong>$:_('You are not <a href="/account/login" title="Log in now">logged in</a>.</strong> Open Library will record your IP address and include it in this page's publicly accessible edit history. If you <a href="/account/create" title="Create an Open Library account">create an account</a>, your IP address is concealed and you can more easily keep track of all your edits and additions.')</p>
```

Comment:
In general we try not to break the html if possible; we have a test that checks that html is valid after translation.

```suggestion
    <p class="collapse sansserif small brown">$:_('<strong>You are not <a href="/account/login" title="Log in now">logged in</a>.</strong> Open Library will record your IP address and include it in this page\'s publicly accessible edit history. If you <a href="/account/create" title="Create an Open Library account">create an account</a>, your IP address is concealed and you can more easily keep track of all your edits and additions.')</p>
```

Reply:
@cdrini ok, I will apply this to the other instances where I saw the `<strong>` tag

Reply:
@cdrini What about in [these instaces](openlibrary/templates/lib/message_addbook.html) with &nbsp eg:
`<p class="list"><span class="red">*</span>&nbsp;&nbsp;$_("Add an ISBN") <span class="small sansserif brown">&nbsp;$_("Help the library connect with other systems, like Amazon.")</span></p>`

Reply:
Might have answered my own question, please confirm I'm understanding the convention correctly:

If a closing html tag is nested in the i18n parens, include the opening tag in the i18n parens as well. &nbsp and i18n message like these: `  &lt;strong&gt;<strong>$_("This text is bold")</strong>&lt;strong&gt; OR **<strong>$_("This text is also bold")</strong>**</p>`
can be left out of parens.